code/web/index.php

```diff
@@ -233,6 +233,17 @@
 		//This happens before the table is setup
 	}
 }
+//Check to see if we should show the cookieConsent banner
+$interface->assign('cookieStorageConsent', false);
+try {
+	require_once ROOT_DIR . '/sys/SystemVariables.php';
+	$systemVariables = new SystemVariables();
+	if ($systemVariables->find(true) && !empty($systemVariables->cookieStorageConsent)) {
+		$interface->assign('cookieStorageConsent', true);
+		$interface->assign('cookieStorageConsentHTML', $systemVariables->cookiePolicyHTML);
+	}
+} catch (Exception $e) {
```

Comment:
Should we be doing something with this catch?  Displaying an error to the end user, logging the error somewhere in the system logs?